test/e2e/runner/main.go

```diff
@@ -160,6 +178,10 @@ func NewCLI() *CLI {
 
 	cli.root.PersistentFlags().StringP("infrastructure-data", "", "", "path to the json file containing the infrastructure data. Only used if the 'infrastructure-type' is set to a value other than 'docker'")
 
+	cli.root.PersistentFlags().StringP(FlagInfluxDBURL, "", "", "URL of the InfluxDB instance to use for arbitrary data collection. If not specified, data will not be collected")
+
+	cli.root.PersistentFlags().StringP(FlagInfluxDBToken, "", "", "Token to use when writing to the InfluxDB instance. Must be specified if 'influxdb-url' is specified")
```

Comment:
we will have some influx server running somewhere, and we can just create account for all interested team members, then those accounts can create tokens as needed.

Reply:
Can you add these flags to the `cmd/tendermint/..` directory as well